cmd/clusterctl/examples/openstack/provider-component/user-data/centos/templates/master-user-data.sh

```diff
@@ -124,81 +101,17 @@ EOF
 
 # Set up kubeadm config file to pass parameters to kubeadm init.
 cat > /etc/kubernetes/kubeadm_config.yaml <<EOF
-apiVersion: kubeadm.k8s.io/v1beta1
-kind: InitConfiguration
-bootstrapTokens:
-- groups:
-  - system:bootstrappers:kubeadm:default-node-token
-  token: ${TOKEN}
-  ttl: 24h0m0s
-  usages:
-  - signing
-  - authentication
-localAPIEndpoint:
-  bindPort: 443
-nodeRegistration:
-  criSocket: /var/run/dockershim.sock
-  kubeletExtraArgs:
-    cloud-config: /etc/kubernetes/cloud.conf
-    cloud-provider: openstack
-  taints:
-  - effect: NoSchedule
-    key: node-role.kubernetes.io/master
----
-apiVersion: kubeadm.k8s.io/v1beta1
-kind: ClusterConfiguration
-kubernetesVersion: v${CONTROL_PLANE_VERSION}
-apiServer:
-  extraArgs:
-    cloud-config: /etc/kubernetes/cloud.conf
-    cloud-provider: openstack
-  extraVolumes:
-  - hostPath: /etc/kubernetes/cloud.conf
-    mountPath: /etc/kubernetes/cloud.conf
-    name: cloud
-    readOnly: true
-  - hostPath: "/etc/certs/cacert"
```

Comment:
this was missed in userdata/kubeadm.go

Reply:
Do you know why this is needed? We don't need this on-premise, but our registry has a regular certificate

Reply:
are you using https or http ? for me , it's a https environment, and openstack cloud you connected through the cacert,
unfortunately my env just broken so I can't show the error I had, it's actually kube controller container failed to start due to it can't find this file

so if we don't want the machines we created able to talk to openstack cloud, we can avoid this but we need fix somewhere else (I need find it later) to make kube controller able to start, or we need add cacert here ,after all ,it's previously there before this PR..

Reply:
No it's found, I'll add it back. Just wanted to understand for what it's used. I'll try to find out how it's done in our environments.

Reply:
Okay I overlooked it in our on-prem installation because we're using the openstack cloud controller manager there instead.

Reply:
@jichenjc Unfortunately I do not have self-signed keystone environment now.
@sbueringer This cacert is for using self-signed keystone, will be ca-file in cloud.conf.
https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#global
I tested only Ubuntu though.

Reply:
I can paste my test env later, I need cacert definitely otherwise the whole openstack APi can't be called

Reply:
I added the mountpoint to controller-manager config